### DISCUSSION TITLE
Want to rename `forkModeDisallowMaintainerEdits` to `forkModeBlockMaintainerEdits`?

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#24945] [Suggest an Idea]
### Tell us more.

Disallow is not a common word, but _block_ is a common word. How about renaming the config option like this?

```diff
- forkModeDisallowMaintainerEdits
+ forkModeBlockMaintainerEdits
```

Todo (if accepted):

- [ ] Update the `description` for `forkModeDisallowMaintainerEdits`
- [ ] Update docs for `forkModeDisallowMaintainerEdits` config option
- [ ] Update other mentions of the old name
- [ ] Write config migration

I'll let somebody else do the work, if this feature is accepted.

**Comment by rarkins** [comment]
The wording GitHub uses is "Allow edits from maintainers.". So it was intended to map to that, while being a value which defaults to false

**Reply by HonkingGoose** [reply]
Let's stick with GitHub's wording then. 😄
